sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -95,15 +95,15 @@ statement
     : query                                                            #statementDefault
     | ctes? dmlStatementNoWith                                         #dmlStatement
     | USE NAMESPACE? multipartIdentifier                               #use
```

Comment:
Should commands like `USE NAMESPACE` and `SHOW CURRENT NAMESPACE ` utilize `namespace`?

Reply:
For new commands, I think we shouldn't support DATABASE.